modules/setting-up-a-customized-argocd-instance.adoc

```diff
@@ -0,0 +1,80 @@
+// Module is included in the following assemblies:
+//
+// * openshift-docs/cicd/gitops/getting-started--with-openshift-gitops.adoc
+
+[id="setting-up-a-customized-argocd-instance_{context}"]
+= Setting up a customized Argo CD instance
+
+The Red Hat OpenShift GitOps operator automatically installs an instance of Argo CD in the `openshift-gitops` namespace with additional permissions, allowing Argo CD to manage cluster-scoped resources.
+
+If you want to install your own instance for managing your cluster configurations or deploying your applications, you must deploy your own instance of Argo CD. Using your own instance of Argo CD helps you to continuously deliver your applications.
+
+.Installing ArgoCD
+
+To install an Argo CD instance:
+
+. In the *Administrator* perspective of the web console, navigate to *Operators* -> *Installed Operators*.
+. Create a new project or select an existing project, depending on where you want to install the Argo CD instance.
+. In the *Installed Operators* page, select *OpenShift GitOps* and then the *Argo CD* tab.
+. Create an Argo CD instance and specify the following configurations:
+*** Name: <argocd-instance-name>
+*** Server > Route > Enable Route = true. This configuration creates an external open source (OS) route to access the Argo CD server.
+
+.Deploying resources to a different namespace
+
+To grant Argo CD the permissions to manage resources in multiple namespaces, create role bindings which provide Argo CD admin permissions for that particular namespace. Use the example, `OOTB Argo CD` instance, deployed in the `openshift-gitops` namespace.
+
+To create a role binding:
+
+. In the *Administrator* perspective of the web console, navigate to *User Management* -> *Role Bindings* > select *Create Binding*. You now see a form that guides you through the process of creating a role binding:
```

Comment:
select *Create Binding* either remove select from this or remove the arrow and add 'and then select..'
arrow needs to be -> in case you retain it.